src/react-components/media-browser.js

```diff
@@ -195,6 +196,7 @@ class MediaBrowser extends Component {
     const urlSource = searchParams.get("media_source") || sluglessPath(this.props.history.location).substring(7);
     const apiSource = this.state.result && this.state.result.meta.source;
     const isVariableWidth = this.state.result && ["bing_images", "tenor"].includes(apiSource);
+    const showCustomOption = apiSource !== "scene_listings" || this.props.hubChannel.permissions.update_hub;
```

Comment:
Maybe not in this PR but I was thinking we should have more explicit API for permissions check. A `permissions.can("update_hub")` method would allow us to validate permission names and warn on typos or name changes. (though I suppose we could use a Proxy if we wanted to get fancy)

Reply:
Also wonder if we should just start using a `window.APP.permissions` global at this point, instead of continuing to pass a hubChannel prop everywhere.

Reply:
i was using the A-Frame system outside of React and using hubChannel -- I agree a global API may make sense.